### PR TITLE
OV settings task resource

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Describe what this change achieves]

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG file.

### CODE REVIEW

File: hpeOneView/resources/activity/tasks.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class Tasks(ResourcePatchMixin, Resource):
     def __init__(self, connection, data=None):
```
Comment (yuvirani) [inline hpeOneView/resources/activity/tasks.py:38]: Add childLimit and topCount in Document's Args section.

File: examples/tasks.py [hunk 2]
```diff
@@ -21,10 +21,10 @@
 from config_loader import try_load_from_file
 
 config = {
-    "ip": "<oneview_ip>",
+    "ip": "",
```
Comment (yuvirani) [inline examples/tasks.py:24]: Change Copyright for all

File: hpeOneView/resources/activity/tasks.py (hunk 1 above)
Comment (nabhajit-ray) [reply hpeOneView/resources/activity/tasks.py:38]: Done

File: examples/tasks.py (hunk 2 above)
Comment (nabhajit-ray) [reply examples/tasks.py:24]: Done

**Review by yuvirani — APPROVED** [review]

File: hpeOneView/resources/resource.py [hunk 3]
```diff
@@ -369,7 +369,8 @@ def __init__(self, base_uri, connection, task_monitor):
         self._task_monitor = task_monitor
 
     def get_all(self, start=0, count=-1, filter='', query='', sort='', view='', fields='', uri=None, scope_uris='', custom_headers=None,
-                name_prefix='', category=[]):
+                name_prefix='', category=[], childLimit=0, topCount=0,):
```
Comment (SHANDCRUZ) [inline hpeOneView/resources/resource.py:372]: there is an extra comma appended at the end (after topCount)

**Review by SHANDCRUZ — APPROVED** [review]

**Review by VenkateshRavula — APPROVED** [review]
Add changelog and endpoint-support file

**Pull request merged**